src/components/paintingCard/PaintingCard.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react'
+import './PaintingCard.css'
+
+const PaintingCard = () => {
+  return (
+      <section className='individual-painting' id={PaintingCard.id}>
```

Comment:
you should use `object destructuring`.  this would clean up the code a little.
[Learn more here](https://www.educative.io/answers/what-is-object-destructuring-in-javascript)